scene/resources/audio_stream_wav.cpp

```diff
@@ -651,8 +651,7 @@ Error AudioStreamWAV::save_to_wav(const String &p_path) {
 	file->store_32(sub_chunk_2_size); //Subchunk2Size
 
 	// Add data
-	Vector<uint8_t> stream_data = get_data();
-	const uint8_t *read_data = stream_data.ptr();
+	const uint8_t *read_data = data.ptr() + DATA_PAD;
```

Comment:
Now there are two places that need to know about `DATA_PAD`, can you think of a way to do this where this is a bit less brittle?

Reply:
`get_data()` by design returns a new data array with the pad removed. As the change gets the data directly, adding `DATA_PAD` to the pointer is necessary.

Reply:
I understand why it is necessary, what I'm objecting to is spreading the knowledge of `DATA_PAD` further into the class :) This is not an existential problem for this PR, but I was hoping you had some idea perhaps on how to prevent this from leaking further.

Reply:
Well, having to add the `DATA_PAD` is inevitably a consequence of moving away from `get_data()`. I don't see any other way around.

Reply:
> The best way to do this, honestly, would be reworking AudioStreamWAV entirely to remove the need of a `DATA_PAD`.

On the subject of rework, there is #100652, which effectively removes the need of a `DATA_PAD`.